tasks/gen/action.cr

```diff
@@ -60,7 +33,35 @@ class Gen::Action < LuckyCli::Task
   end
 
   private def name_matches_format
-    @error = "That's not a valid Action.  Example: lucky gen.action Users::Index"
+    @error = "That's not a valid Action. Example: lucky gen.action Users::Index"
     ARGV.first.includes?("::")
   end
+
+  private def action_name
+    ARGV.first
+  end
+
+  private def action
+    path_args.last
+  end
+
+  private def output_path
+    Dir.current + app_directory_path
+  end
+
+  private def app_directory_path
+    "/src/actions/#{path}"
+  end
+
+  private def path
+    path_args[0..-2].join("/")
+  end
+
+  private def path_args
+    action_name.split("::").map(&.downcase)
+  end
+
+  private def success_message
+    "Done generating #{action_name.colorize(:green)} in #{app_directory_path.colorize(:green)}"
```

Comment:
Can you add a `.` before the `app_directory_path` so it's clear it's not an absolute path?

Reply:
💯